crates/db_schema/src/source/site.rs

```diff
@@ -34,7 +34,9 @@ pub struct Site {
   pub last_refreshed_at: DateTime<Utc>,
   /// The site inbox
   pub inbox_url: DbUrl,
+  #[serde(skip)]
   pub private_key: Option<String>,
+  #[serde(skip)]
```

Comment:
Doesn't the pub key need to get sent elsewhere? I guess the federation tests would be failing if that weren't the case.

Reply:
Yes but that is done in a separate struct. You can compare the db structs for user or community which have the same skip attributes.